ajax/libs/leaflet-realtime/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "leaflet-realtime",
+  "filename": "leaflet-realtime.min.js",
+  "npmName": "leaflet-realtime",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "version": "1.3.0",
+  "description": "Show realtime updated GeoJSON in Leaflet",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/perliedman/leaflet-realtime.git"
+  },
+  "keywords": [
+    "leaflet",
+    "realtime"
+  ],
+  "author": {
+    "name": "Per Liedman",
+    "email": "per@liedman.net"
+  },
+  "license": "ISC",
+  "browserify-shim": {
+    "leaflet": "global:L"
+  },
```

Comment:
Thanks @perliedman, in fact we don't manually build libraries, we only host pre-built files from upstream, thanks!